sites/default/conf/google_analytics.settings.yml

```diff
@@ -0,0 +1,41 @@
+_core:
```

Comment:
Google analytics should be disabled sooner or later. We're not checking GA and we're adding some tracking to the web.